<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="alp22060" failures="2" tests="2" name="RetryMultipleFailedTestCases" time="0.056" errors="0" timestamp="2024-06-13T16:21:43 IST">
  <testcase classname="mytests.MyTests" name="Test1" time="0.013">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at mytests.MyTests.Test1(MyTests.java:9)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- Test1 -->
  <testcase classname="mytests.MyTests" name="Test2" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at mytests.MyTests.Test2(MyTests.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- Test2 -->
</testsuite> <!-- RetryMultipleFailedTestCases -->
